src/Xamarin.Android.Build.Tasks/Utilities/MarshalMethodsNativeAssemblyGenerator.cs

```diff
@@ -171,6 +171,9 @@ sealed class MarshalMethodName
 			{ 'L', typeof(_jobjectArray) },
 		};
 
+		const string mm_trace_func_enter_name = "_mm_trace_func_enter";
```

Comment:
Seems *kinda* odd that most of the members here are camelCase or PascalCase, but then the new members are `c_snake_case`.  Should this *not* be `MMTraceFuncEnterName`?

Reply:
It's a standalone C/C++ function, so I used the C/C++ naming convention.